src/Common/Common.Api/Gateways/GatewayClient.cs

```diff
@@ -188,6 +188,134 @@ public async Task<HttpResponseMessage> UpdateInstallerPrincipals(UpdateGatewayIn
             return response;
         }
 
+        public async Task<IEnumerable<GatewayClusterDatasource>> GetGatewayClusterDatasources(Guid gatewayClusterId, bool asIndividual)
+        {
+            var url = Invariant($"{GetODataUrlStart(true)}/gatewayClusters/{gatewayClusterId}/datasources");
```

Comment:
use  {GetODataUrlStart(asIndividual)} instead of {GetODataUrlStart(true)}. Same as other place.

Reply:
good point!